### PR TITLE
Add `os_codename` variable

### PR DESCRIPTION
**Pull request by hrpatel**
- Add an `os_code` variable
- Replace tabs with spaces

### CODE REVIEW

File: shtdlib.sh [hunk 1]
```diff
@@ -60,7 +61,7 @@ os_type="$(uname)"
 
 # Determine virtualization platform in a way that ignores SIGPIPE, requires root
 if [ "${EUID}" == 0 ] && command -v virt-what &> /dev/null ; then
-    virt_platform="$(virt-what | head -1 || if [[ ${?} -eq 141 ]]; then true; else exit ${?}; fi)"
+    virt_platform=$(virt-what | head -1 || if [[ ${?} -eq 141 ]]; then true; else exit ${?}; fi)
```
Comment (antevens) [inline shtdlib.sh]: Any reason to remove the quotes?

File: shtdlib.sh [hunk 2]
```diff
@@ -84,10 +85,13 @@ if [ "${os_family}" == 'RedHat' ]; then
         os_name='redhat';
     fi
 elif [ "${os_family}" == 'Debian' ]; then
+    os_codename="$(grep UBUNTU_CODENAME /etc/os-release | awk -F= '{print $2}')"
```
Comment (antevens) [inline shtdlib.sh]: Do we need to handle any cases where this file does not exist or is empty?

File: shtdlib.sh (hunk 1 above)
Comment (hrpatel) [reply shtdlib.sh]: Disregard.. that change should not have been committed

File: shtdlib.sh (hunk 2 above)
Comment (hrpatel) [reply shtdlib.sh]: Updated

**Pull request merged**